AppStub/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.2.1</string>
+	<string>$(MARKETING_VERSION)</string>
```

Comment:
Oh, fancy ...

Reply:
The bin/git-tag.sh script reads this Info.plist for the DeviceAgent tag.

https://github.com/calabash/DeviceAgent.iOS/blob/develop/bin/git-tag.sh#L5

Rollback this change or update the script to read the .xcodeproj file (hard) and update the documentation.